src/uri/serializer/microuriserializer.rs

```diff
@@ -268,7 +327,7 @@ mod tests {
         assert!(uprotocol_uri.is_err());
         assert_eq!(
             uprotocol_uri.unwrap_err().to_string(),
-            "URI is empty or not in micro form"
+            "Failed to validate micro URI format: URI is empty"
```

Comment:
I'd rather remove these literal comparisons as they are hard to maintain. In particular, when not using constants for the messages ...

Reply:
Yeah, I could go through this file and remove these.

To a degree I was following along with the rest of the unit tests.

You want me to remove them all from this file?

Reply:
Okay, went ahead and did that, could you take another look?